tests/console/yast2_rmt.pm

```diff
@@ -84,5 +84,11 @@ sub run {
     assert_script_run("firewall-cmd --remove-service=http{,s}");
 }
 
+sub post_fail_hook {
+    my ($self) = @_;
+    select_console 'log-console';
+    $self->export_logs_basic;
```

Comment:
I've checked and the only things which are not collected are `/proc/loadavg` and `/etc/sysconfig`. dmesg, journal and ps output will be collected twice. Do you think it can be useful for other scenarios too? If yes, I would add it to `y2_module_basetest` directly. If not, let's merge this for rmt module only.